resources/images/s3/filestorage.go

```diff
@@ -54,10 +54,21 @@ func NewSimpleStorageServiceStatic(bucket, key, secret, region, token, uri strin
 		config = config.WithDisableSSL(sslDisabled).WithEndpoint(uri)
 	}
 
+	config.S3ForcePathStyle = aws.Bool(true)
 	sess := session.New(config)
 
+	client := s3.New(sess)
+
+	// minio requires explicit bucket creation
+	cparams := &s3.CreateBucketInput{
+		Bucket: aws.String(bucket), // Required
+	}
+
+	// Create a new bucket using the CreateBucket call.
+	client.CreateBucket(cparams)
```

Comment:
erring up I think in case of issues creating bucket is OK, we do that in some other cases 

staring service with bucket already existing is going to be pretty common case:
* starting second, third etc instance in setup
* manually setup bucket in S3
* service restarts

create if not exists sounds like a good idea, we don't need to require it as precondition